src/mixins/amalgamation-mixin.ts

```diff
@@ -0,0 +1,114 @@
+import { Component, Vue } from 'vue-property-decorator'
+import { Getter } from 'pinia-class'
+import { useStore } from '@/store/store'
+import { AmalgamatingStatuses } from '@/enums'
+import { AmalgamatingBusinessIF } from '@/interfaces'
+import { CorpTypeCd } from '@bcrs-shared-components/corp-type-module'
+
+/**
+ * Mixin that provides amalgamation rules, etc.
+ */
+@Component({})
+export default class AmalgamationMixin extends Vue {
+  @Getter(useStore) getAmalgamatingBusinesses!: AmalgamatingBusinessIF[]
+  @Getter(useStore) isRoleStaff!: boolean
+  @Getter(useStore) isTypeBcCcc!: boolean
+  @Getter(useStore) isTypeBcUlcCompany!: boolean
+
+  // *** TODO: finish this, maybe name them something recognizable...
```

Comment:
WIP but some functionality is in and usable. I will pick away at this but I am no longer blocking anyone.